iOS/Multiplatform Redux Sample/Podfile

```diff
@@ -5,8 +5,8 @@ target 'Multiplatform Redux Sample' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
 
-  # pod 'dreiKit', :path => '../../../dreiKit'
-  pod 'dreiKit', :git => 'https://github.com/dreipol/dreiKit', :tag => '1.0.3'
+  pod 'dreiKit', :path => '../../../dreiKit'
+  # pod 'dreiKit', :git => 'https://github.com/dreipol/dreiKit', :tag => '1.0.3'
```

Comment:
Can we use the 1.0.4 release for this? If so, please update.

Reply:
Yes, I will.